src/berry.c

```diff
@@ -807,15 +807,15 @@ const struct Berry gBerries[] =
 // unused
 void ClearEnigmaBerries(void)
 {
-    CpuFill16(0, &gSaveBlock1Ptr->enigmaBerry, sizeof(gSaveBlock1Ptr->enigmaBerry));
+    CpuFill16(0, &gSaveBlock1Ptr->enigmaBerry, 52 /*FIXME: sizeof(gSaveBlock1Ptr->enigmaBerry)*/);
```

Comment:
This has already been fixed in an earlier PR so you can remove the FIXME sections.